packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -903,13 +904,14 @@ function detachFiber(current: Fiber) {
   current.memoizedState = null;
   current.updateQueue = null;
   current.dependencies = null;
-  const alternate = current.alternate;
+  current.sibling = null;
+  current.alternate = null;
```

Comment:
I believe this shouldn’t matter to set.

Reply:
It's not needed but it reduces the amount of code quite a bit by re-using the function (which should be hot). I can go back to the previous way if you feel it's better.